<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>maintainBudget</class>
 <widget class="QWidget" name="maintainBudget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maintain Budget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <layout class="QGridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="_descripLit">
         <property name="text">
          <string>Description:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="_descrip"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="_name"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="_nameLit">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox1">
       <property name="title">
        <string>Table Parameters</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="_accountsLit">
              <property name="text">
               <string>Accounts:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="XTreeWidget" name="_accounts">
                <property name="alternatingRowColors">
                 <bool>false</bool>
                </property>
                <property name="rootIsDecorated">
                 <bool>false</bool>
                </property>
                <attribute name="headerVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout">
                <item>
                 <widget class="QPushButton" name="_accountsAdd">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_accountsRemove">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::MinimumExpanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="_periodsLit">
              <property name="text">
               <string>Accounting Periods:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QListWidget" name="_periods">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout">
                <item>
                 <widget class="QPushButton" name="_periodsAll">
                  <property name="text">
                   <string>Select All</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_periodsInvert">
                  <property name="text">
                   <string>Invert</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_periodsNone">
                  <property name="text">
                   <string>Unselect All</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="_generate">
            <property name="text">
             <string>Generate Table</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_save">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_print">
       <property name="text">
        <string>&amp;Print</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTableWidget" name="_table">
        <property name="numRows" stdset="0">
         <number>0</number>
        </property>
        <property name="numCols" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_name</tabstop>
  <tabstop>_descrip</tabstop>
  <tabstop>_accounts</tabstop>
  <tabstop>_accountsAdd</tabstop>
  <tabstop>_periods</tabstop>
  <tabstop>_periodsAll</tabstop>
  <tabstop>_periodsInvert</tabstop>
  <tabstop>_periodsNone</tabstop>
  <tabstop>_generate</tabstop>
  <tabstop>_table</tabstop>
  <tabstop>_close</tabstop>
  <tabstop>_save</tabstop>
  <tabstop>_print</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>maintainBudget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_periodsNone</sender>
   <signal>clicked()</signal>
   <receiver>_periods</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
